internal/cmd/update.go

```diff
@@ -0,0 +1,108 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/sonatype-nexus-community/nancy/buildversion"
+	"github.com/sonatype-nexus-community/nancy/update"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(updateCmd)
+}
+
+var updateCmd = newUpdateCommand()
+
+func newUpdateCommand() *cobra.Command {
+
+	updateCmd := &cobra.Command{
+		Use: "update",
+		//Short: "Update the tool to the latest version",
+		Short: "Check if there are any updates available",
+		RunE: func(_ *cobra.Command, _ []string) error {
+			//return updateCLI("", true)
+			return updateCLI("", false)
+		},
+	}
+
+	/*	updateCmd.AddCommand(&cobra.Command{
+			Use:    "check",
+			Hidden: true,
+			Short:  "Check if there are any updates available",
+			RunE: func(_ *cobra.Command, _ []string) error {
+				return updateCLI("", false)
+			},
+		})
+	*/
```

Comment:
So why the added `check` here?? Is this the future?? since there is also an `install` commented out below??